bonita-integration-tests/bonita-integration-tests-client/src/main/java/org/bonitasoft/engine/business/data/BDRepositoryIT.java

```diff
@@ -1247,6 +1246,40 @@ public void useMultipleBusinessDataInACallActivityWithOutDataMultiInstance() thr
         disableAndDeleteProcess(subProcessDefinition);
     }
 
+
+    //BS-13803
+    @Test
+    public void initializeBusinessDataInCalledProcessWithContractInput() throws Exception {
+        final Expression employeeExpression = new ExpressionBuilder().createGroovyScriptExpression(
+                "createNewEmployee",
+                "import " + EMPLOYEE_QUALIFIED_NAME + "; Employee john = new Employee(); john.firstName = theInput; john.lastName = 'Doe'; john;",
+                EMPLOYEE_QUALIFIED_NAME, new ExpressionBuilder().createContractInputExpression("theInput", String.class.getName()));
+        ProcessDefinitionBuilder builder = new ProcessDefinitionBuilder().createNewInstance("createEmployeeInCallActivity", "1.2-beta");
+        builder.addActor(ACTOR_NAME);
+        builder.addContract().addInput("theInput", Type.TEXT, "the input");
+        builder.addBusinessData("employee", EMPLOYEE_QUALIFIED_NAME, employeeExpression);
+        builder.addUserTask("step1", ACTOR_NAME);
+        final ProcessDefinition subProcessDefinition = deployAndEnableProcessWithActor(builder.done(), ACTOR_NAME, matti);
+
+        builder = new ProcessDefinitionBuilder().createNewInstance("createEmployeeInCallActivityMaster", "1.2-beta");
+        builder.addActor(ACTOR_NAME);
+        builder.addCallActivity("call",
+                new ExpressionBuilder().createConstantStringExpression(subProcessDefinition.getName()),
+                new ExpressionBuilder().createConstantStringExpression(subProcessDefinition.getVersion())).addProcessStartContractInput("theInput", new ExpressionBuilder().createConstantStringExpression("theValue"));
+        builder.addUserTask("step2", ACTOR_NAME);
+        builder.addTransition("call", "step2");
+        final ProcessDefinition processDefinition = deployAndEnableProcessWithActor(builder.done(), ACTOR_NAME, matti);
+
+        final ProcessInstance instance = getProcessAPI().startProcess(processDefinition.getId());
+        final long step1Id = waitForUserTask("step1");
+        getProcessAPI().assignUserTask(step1Id, matti.getId());
+        getProcessAPI().executeFlowNode(step1Id);
+        waitForUserTask(instance, "step2");
+
+        disableAndDeleteProcess(processDefinition);
+        disableAndDeleteProcess(subProcessDefinition);
```

Comment:
kind of, we are checking that the step 2 is available.

I can add some more assertions